script.js

```diff
@@ -1,12 +1,235 @@
-//You can edit ALL of the code here
+"use strict";
+const rootElem = document.getElementById("root");
+const mainElement = document.createElement("main");
+const selectEpisodeElement = document.getElementById("episode-select");
+const selectEpisodesLabel = document.getElementById("episode-select-label");
+const selectShowElement = document.getElementById("show-select");
+const inputElement = document.querySelector("input");
+const resetButton = document.getElementById("reset");
+let searchedEpisode = document.getElementById("search-result");
+let allEpisodes = document.getElementById("total-episodes");
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  reset();
+  selectShow();
+  displayShow();
+  searchBox("shows");
+}
+
+function addZero(x) {
+  if (x < 10) {
```

Comment:
Can you think of a way to shorten this to a [ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)?